src/lib/ibcclient.ts

```diff
@@ -1339,37 +1336,23 @@ export class IbcClient {
     token: Coin,
     receiver: string,
     timeoutHeight?: Height,
-    // timeout in seconds (we make nanoseconds below)
+    /** timeout in seconds (SigningStargateClient converts to nanoseconds) */
     timeoutTime?: number
   ): Promise<MsgResult> {
     this.logger.verbose(`Transfer tokens to ${receiver}`);
-    const senderAddress = this.senderAddress;
-    const timeoutTimestamp = timeoutTime
-      ? Long.fromNumber(timeoutTime).multiply(1_000_000_000)
-      : undefined;
-    const msg = {
-      typeUrl: '/ibc.applications.transfer.v1.MsgTransfer',
-      value: MsgTransfer.fromPartial({
-        sourcePort,
-        sourceChannel,
-        sender: senderAddress,
-        token,
-        receiver,
-        timeoutHeight,
-        timeoutTimestamp,
-      }),
-    };
-    this.logger.debug('MsgTransfer', msg);
```

Comment:
Is this log message relevant?

Reply:
See https://github.com/confio/ts-relayer/pull/150#issuecomment-814881096